pwndbg/ghidra.py

```diff
@@ -19,25 +19,25 @@
 
 from pwndbg.color import message
 
-r2decompiler = pwndbg.config.add_param(
-    "r2decompiler",
+decompiler = pwndbg.config.add_param(
+    "decompiler",
     "radare2",
     "framework that your ghidra plugin installed",
     param_class=pwndbg.lib.config.PARAM_ENUM,
     enum_sequence=["radare2", "rizin"],
 )
 
 
-@pwndbg.config.trigger(r2decompiler)
-def set_r2decompiler() -> None:
-    if r2decompiler.value in ["radare2", "rizin"]:
+@pwndbg.config.trigger(decompiler)
+def set_decompiler() -> None:
+    if decompiler.value in ["radare2", "rizin"]:
         return
     print(
         message.warn(
-            f"Invalid r2decompiler: `{r2decompiler.value}`, please select from radare2 or rizin"
+            f"Invalid decompiler: `{decompiler.value}`, please select from radare2 or rizin"
         )
     )
-    r2decompiler.revert_default()
+    decompiler.revert_default()
```

Comment:
Btw is this trigger needed at all considering that we have a PARAM_ENUM and `enum_sequence=["radare2", "rizin"],`? Isn't this config param providing such handling by itself? (I don't know, but I believe it rather should :P)